aws/scenarios/microVMs/microvms/run.go

```diff
@@ -149,7 +149,10 @@ func configureInstance(instance *Instance, m *config.DDMicroVMConfig) ([]pulumi.
 
 	env := *instance.e.CommonEnvironment
 	osCommand := command.NewUnixOSCommand()
-	localRunner := command.NewLocalRunner(env, osCommand)
+	localRunner, err := command.NewLocalRunner(env, osCommand)
+	if err != nil {
+		return waitFor, err
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
`waitFor` is probably empty, following other error handlings you should return `nil, err
```suggestion
		return nil, err
```